Alluvial.Tests/Alluvial.Tests.csproj

```diff
@@ -86,7 +101,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="StreamQueryTests.cs" />
     <Compile Include="TracingTests.cs" />
-    <Compile Include="Wait.cs" />
+    <Compile Include="%28Infrastructure%29\Wait.cs" />

```

Comment:
Not a big fan of parenthesis as I can't just copy and paste them from a `git status`